site/config.toml

```diff
@@ -7,6 +7,10 @@ publishDir = "../docs"
 pygmentsCodeFences = true
 pygmentsStyle = "manni"
 
+[taxonomies]
+  category = "categories"
+  tag = "tags"
```

Comment:
Are tags used anywhere?

Reply:
Nope, but this is just Hugo's default. I'll remove it then.